Mathlib/Data/Fin/Tuple/Basic.lean

```diff
@@ -4,8 +4,10 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Floris van Doorn, Yury Kudryashov, Sébastien Gouëzel, Chris Hughes
 -/
 import Mathlib.Data.Fin.Basic
+import Mathlib.Data.Int.Lemmas
 import Mathlib.Data.Pi.Lex
 import Mathlib.Data.Set.Intervals.Basic
+import Mathlib.Tactic.Zify
```

Comment:
How many filed do these 2 imports add to the closure?

Reply:
Not sure I'm looking at the right thing, but the graph produced by `lake exe graph --to Mathlib.Data.Fin.Tuple.Basic --reduce` went from 560 edges to 644.

Reply:
I pushed a proof that doesn't use `zify`. I guess, now we need a review from another maintainer.
maintainer merge